tests/system/action/meeting/test_delete.py

```diff
@@ -144,9 +144,11 @@ def test_delete_full_meeting(self) -> None:
         for i in range(3):
             user = self.get_model(f"user/{i+1}")
             for field in User().get_fields():
-                if isinstance(
-                    field, BaseTemplateField
-                ) and field.replacement_collection == Collection("meeting"):
+                if (
+                    isinstance(field, BaseTemplateField)
+                    and field.replacement_collection
+                    and field.replacement_collection == Collection("meeting")
```

Comment:
what purpose serves this?

Reply:
Could be there is no replacement_collection, but a replacement_enum. With change of eq-operator to try/except construction in this case there will be an exception. Before there has been a silent `False`.

Reply:
but here is no try/except :D you just added `and field.replacement_collection` which I would not think to make any difference (except maybe that the `Collection("meeting")` is not executed?)

Reply:
I still don't understand what you mean, but it's not important